src/hotspot/share/compiler/disassembler.cpp

```diff
@@ -837,8 +836,6 @@ bool Disassembler::load_library(outputStream* st) {
   if (_library != nullptr) {
     _decode_instructions_virtual = CAST_TO_FN_PTR(Disassembler::decode_func_virtual,
                                           os::dll_lookup(_library, decode_instructions_virtual_name));
-  } else {
-    log_warning(os)("Loading hsdis library failed");
```

Comment:
Personally, I would leave this warning. It does not hurt, and perhaps someone is depending on it to detect if hsdis is installed correctly.